upgrade_test.py

```diff
@@ -371,15 +368,6 @@ def _rollback_node(self, node, upgrade_sstables=True):
         node.run_nodetool("snapshot")
         node.stop_scylla_server(verify_down=False)
 
-        if self.params.get("enable_tablets_on_upgrade"):
-            with node.remote_scylla_yaml() as scylla_yml:
-                current_experimental_features = scylla_yml.experimental_features
-                current_experimental_features.remove("tablets")
```

Comment:
Should we instead edit scylla.yaml to remove the newly introduced `enable_tablets` config option?
(See scylladb/scylladb#18898)